x-pack/plugins/observability/public/components/shared/exploratory_view/configurations/infra_metrics/kpi_over_time_config.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ConfigProps, SeriesConfig } from '../../types';
+import { FieldLabels, OPERATION_COLUMN, REPORT_METRIC_FIELD, ReportTypes } from '../constants';
+import {
+  SYSTEM_CPU_PERCENTAGE_FIELD,
+  DOCKER_CPU_PERCENTAGE_FIELD,
+  K8_POD_CPU_PERCENTAGE_FIELD,
+  SYSTEM_MEMORY_PERCENTAGE_FIELD,
+} from '../constants/field_names/infra_metrics';
+import {
+  DOCKER_CPU_USAGE,
+  K8_POD_CPU_USAGE,
+  SYSTEM_CPU_USAGE,
+  SYSTEM_MEMORY_USAGE,
+} from '../constants/labels';
+
+export function getMetricsKPIConfig({ indexPattern }: ConfigProps): SeriesConfig {
+  return {
+    reportType: ReportTypes.KPI,
+    defaultSeriesType: 'area',
+    seriesTypes: [],
+    xAxisColumn: {
+      sourceField: '@timestamp',
+    },
+    yAxisColumns: [
+      {
+        sourceField: REPORT_METRIC_FIELD,
+        operationType: 'median',
+      },
+    ],
+    hasOperationType: false,
+    filterFields: ['agent.type', 'service.type'],
+    breakdownFields: ['agent.hostname', 'service.type'],
+    baseFilters: [],
+    definitionFields: ['agent.hostname', 'service.type'],
+    metricOptions: [
+      {
+        label: SYSTEM_CPU_USAGE,
+        field: SYSTEM_CPU_PERCENTAGE_FIELD,
+        id: SYSTEM_CPU_PERCENTAGE_FIELD,
+        columnType: OPERATION_COLUMN,
+      },
+      {
+        label: SYSTEM_MEMORY_USAGE,
+        field: SYSTEM_MEMORY_PERCENTAGE_FIELD,
+        id: SYSTEM_MEMORY_PERCENTAGE_FIELD,
+        columnType: OPERATION_COLUMN,
+      },
+      {
+        label: DOCKER_CPU_USAGE,
+        field: DOCKER_CPU_PERCENTAGE_FIELD,
+        id: DOCKER_CPU_PERCENTAGE_FIELD,
+        columnType: OPERATION_COLUMN,
+      },
+      {
+        label: K8_POD_CPU_USAGE,
+        field: K8_POD_CPU_PERCENTAGE_FIELD,
+        id: K8_POD_CPU_PERCENTAGE_FIELD,
+        columnType: OPERATION_COLUMN,
+      },
+    ],
+    labels: { ...FieldLabels },
```

Comment:
Is this spread in order to prevent clobbering `FieldLabels` by the receiver of config?